packages/superset-ui-legacy-preset-chart-nvd3/test/utils.test.js

```diff
@@ -72,12 +72,12 @@ const DATA_WITH_DISABLED_SERIES = [
     key: ['East Asia & Pacific'],
     values: [
       {
-        x: -315619200000.0,
```

Comment:
Not sure if there was a valid reason these are `.0` like to cast as float and make sure it comes through a formatter correctly.

Reply:
Added `eslint-disable` for the entire file and bring back `.0`